From 58825f74eb0156822065c449a770644a69044d88 Mon Sep 17 00:00:00 2001
From: Matthias Brugger <mbrugger@suse.com>
Date: Tue, 16 Feb 2021 20:35:23 +0100
Subject: [PATCH] brcm: Fix Raspberry Pi 4B NVRAM file

upstream repo git://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git
upstream commit 58825f74eb0156822065c449a770644a69044d88

NVRAM file set's boarflags3 to 0x48200100 which indicates the firmware
that an external LPO is present. But on the RPi4 no external LPO exists.

Fixes: 870b805 ("brcm: Update Raspberry Pi 3B+/4B NVRAM for downstream changes")
Signed-off-by: Matthias Brugger <mbrugger@suse.com>
Signed-off-by: Josh Boyer <jwboyer@kernel.org>
---
 brcm/brcmfmac43455-sdio.raspberrypi,4-model-b.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/brcm/brcmfmac43455-sdio.raspberrypi,4-model-b.txt b/brcm/brcmfmac43455-sdio.raspberrypi,4-model-b.txt
index d5bd530..8dda168 100644
--- a/brcm/brcmfmac43455-sdio.raspberrypi,4-model-b.txt
+++ b/brcm/brcmfmac43455-sdio.raspberrypi,4-model-b.txt
@@ -7,7 +7,7 @@ aa5g=1
 AvVmid_c0=1,165,2,100,2,100,2,100,2,100
 boardflags=0x00480201
 boardflags2=0x40800000
-boardflags3=0x48200100
+boardflags3=0x44200100
 boardrev=0x1304
 boardtype=0x6e4
 btc_mode=1
-- 
2.17.1

